authorization-service/src/libs/lambda.ts

```diff
@@ -0,0 +1,6 @@
+import middy from "@middy/core";
+import middyJsonBodyParser from "@middy/http-json-body-parser";
+
+export const middyfy = (handler) => {
+    return middy(handler).use(middyJsonBodyParser());
```

Comment:
no need to use middleware here at all. As I've understood - authorizer is as middleware by itself and middyJsonBodyParser is redundant